arex-instrumentation-api/src/main/java/io/arex/inst/extension/ModuleDescription.java

```diff
@@ -32,12 +36,14 @@ public boolean isSupported(ComparableVersion current) {
     }
 
     public static final class Builder {
-        private String name;
+        private final Set<String> names = new HashSet<>();
```

Comment:
NR: not recommended to initialize it directly, the name parameter is not passed every time